h2-storage/pom.xml

```diff
@@ -33,6 +33,21 @@
 			<artifactId>javax.persistence-api</artifactId>
 			<version>2.2</version>
 		</dependency>
+		<dependency>
+			<groupId>org.hibernate</groupId>
```

Comment:
Do we need this?

Reply:
I just forgot to remove this after I tried to make the Id auto generated. I'm going to fix it